src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/LocalOfferDetail.cshtml.cs

```diff
@@ -37,11 +40,39 @@ public async Task<IActionResult> OnGetAsync(string serviceId)
         ReturnUrl = StringValues.IsNullOrEmpty(referer) ? Url.Page("Search") : referer.ToString();
         LocalOffer = await _organisationClientService.GetLocalOfferById(serviceId);
 
+        (ServiceScheduleAttendingTypes, ServiceSchedule) = GetServiceSchedule();
+
         ShowConnectionRequestButton = await ShouldShowConnectionRequestButton();
 
         return Page();
     }
 
+    // this only covers scenarios that can be created through Manage, not all possible scenarios from LA ingested data
+
+    private (List<AttendingType>, ScheduleDto?) GetServiceSchedule()
+    {
+        var serviceScheduleAttendingTypes = new List<AttendingType>();
+
+        foreach (var attendingTypeString in LocalOffer.Schedules
+                     .Select(s => s.AttendingType))
+        {
+            if (attendingTypeString != null
+                && Enum.TryParse<AttendingType>(attendingTypeString, out var attendingType)
+                && (attendingType != AttendingType.InPerson

```

Comment:
Same as above

Reply:
Just style and != is terser 😄

Reply:
No worries. Just an FYI as objects can override != etc and change the meaning of !=. Happy to close

Reply:
Thanks Steve - remember Enums are sealed, so can't have `!=` overridden.